ci/actions-templates/linux-builds-template.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
           - armv7-unknown-linux-gnueabihf
           - aarch64-linux-android
           - aarch64-unknown-linux-gnu     # skip-pr
+          - aarch64-unknown-linux-musl    # skip-pr
```

Comment:
It's important that this not be skipped on PR until we know it works

Reply:
Done, fixed in 6184ff0